python/src/ngraph/ops.py

```diff
@@ -69,6 +69,21 @@ def constant(value, dtype=None, name=None):  # type: (NumericData, NumericType,
     return make_constant_node(value, dtype)
 
 
+@nameable_op
+def ctc_greedy_decoder(data, sequence_mask, merge_repeated=True, name=None):
+    # type: (Node, Node, bool, str) -> Node
```

Comment:
IMHO it would be convenient if the user has possibility to pass ie. `np.ndarray` object. This is handled by `NodeInput` type and would require to call `as_nodes`.

Reply:
Done